mod_reforged/config/z_nested_tooltips.nut

```diff
@@ -1,7 +1,7 @@
 ::Reforged.NestedTooltips <- {
 	Tooltips = {
 		Concept = {
-			Perk = ::MSU.Class.BasicTooltip("Perk", ::Reforged.Mod.Tooltips.parseString("As characters gain levels, they unlock perk points which can be spent to unlock powerful perks. Perks grant a character permanent bonuses or unlock new skills for use. Perks are a central theme in Battle Brothers and are the key to a character\'s specialization.")),
+			Perk = ::MSU.Class.BasicTooltip("Perk", ::Reforged.Mod.Tooltips.parseString("As characters gain levels, they unlock perk points which can be spent to unlock powerful perks. Perks grant a character permanent bonuses or unlock new skills for use. Each time you spend a perk point you also increase the character\'s current [perk tier|Concept.PerkTier] by 1.")),
```

Comment:
In my opinion, better wording would be:
`Each perk point spent increases the character\'s current [perk tier|Concept.PerkTier] by 1.` 
or 
`The character\'s current [perk tier|Concept.PerkTier] increases by 1 each time a perk point is spent.`